test/test_ops.py

```diff
@@ -210,6 +219,10 @@ def get_slice(k, block):
     def test_boxes_shape(self):
         self._helper_boxes_shape(ops.roi_pool)
 
+    def test_jit_boxes_list(self):
+        model = PoolWrapper(ops.RoIPool(output_size=[3, 3], spatial_scale=1.0))
+        torch.jit.script(model)
```

Comment:
Maybe we could also check model execution on list of tensors ?

Reply:
Yes good call. I'll add it now.